dotcom-rendering/src/web/components/KeyEventsCards.tsx

```diff
@@ -0,0 +1,110 @@
+import { css } from '@emotion/react';
+import { textSans, neutral, space, from } from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import { timeAgo } from '@guardian/libs';
+import { decidePalette } from '../lib/decidePalette';
+
+export interface KeyEvent {
+	date: Date;
+	text: string;
+	url: string;
+	format: ArticleFormat;
+}
+
+const linkStyles = css`
+	text-decoration: none;
+
+	&::before {
+		content: '';
+		display: block;
+		position: relative;
+		height: 13px;
+		width: 13px;
+		border-radius: 50%;
+		background-color: ${neutral[46]};
+		margin-bottom: ${space[1]}px;
+		z-index: 2;
+
+		${from.tablet} {
+			height: 15px;
+			width: 15px;
+		}
+	}
+
+	&:hover::before {
+		background-color: ${neutral[0]};
+	}
+`;
+
+const listItemStyles = css`
+	position: relative;
+	padding-bottom: ${space[5]}px;
+	padding-top: ${space[3]}px;
+	padding-right: ${space[5]}px;
+	background-color: ${neutral[97]};
+	list-style: none;
+	display: block;
+	width: 150px;
+
+	${from.desktop} {
+		background-color: ${neutral[93]};
+		width: 180px;
+	}
+
+	&::before {
+		content: '';
+		display: block;
+		position: absolute;
+		border-top: 1px dotted ${neutral[46]};
+		left: 0;
+		right: 0;
+		top: 18px;
+	}
+
+	&:last-child::before {
+		border-top: 0;
+	}
+`;
+
+const textStyles = (palette: Palette) => css`
+	${textSans.small({ fontWeight: 'regular', lineHeight: 'regular' })};
+	color: ${palette.text.keyEvent};
+
+	&:hover {
+		text-decoration: underline;
+	}
+`;
+
+const timeStyles = css`
+	${textSans.xsmall({ fontWeight: 'bold', lineHeight: 'tight' })};
+	color: ${neutral[7]};
+	display: block;
+`;
+
+export const KeyEventsCard = ({ text, date, url, format }: KeyEvent) => {
```

Comment:
I think if the export is `KeyEventCard` and it's only exporting one thing then the filename should also be `KeyEventCard.tsx`?

Reply:
This component handles one key event so `KeyEventCard` feels right to me.